.changeset/selfish-emus-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hahnpro/flow-sdk': patch
+---
+
+added test for correct instantiation for nested function properties
```

Comment:
Any particular reason why you want to release this as a new version? No features or fixes are added